--- CMS2_Class.h	2008-09-07 10:40:25.000000000 -0700
+++ ../Tools/CMS2_Class.h	2008-09-07 09:31:05.000000000 -0700
@@ -175,9 +175,9 @@
 	vector<vector<ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > > >	hyp_other_jets_tq_jet_p4_;
 	TBranch *hyp_other_jets_tq_jet_p4_branch;
 	bool hyp_other_jets_tq_jet_p4_isLoaded;
-  //edm::TriggerResults	TriggerResults_;
-  //TBranch *TriggerResults_branch;
-  //bool TriggerResults_isLoaded;
+	edm::TriggerResults	TriggerResults_;
+	TBranch *TriggerResults_branch;
+	bool TriggerResults_isLoaded;
 	float	gen_met_;
 	TBranch *gen_met_branch;
 	bool gen_met_isLoaded;
@@ -1288,15 +1288,15 @@
 	vector<vector<int> >	hyp_trilep_jets_index_;
 	TBranch *hyp_trilep_jets_index_branch;
 	bool hyp_trilep_jets_index_isLoaded;
-  //vector<reco::CaloMET>	corMetType1Icone5_;
-  //TBranch *corMetType1Icone5_branch;
-  //bool corMetType1Icone5_isLoaded;
-  //vector<reco::CaloMET>	corMetType1Mcone5_;
-  //TBranch *corMetType1Mcone5_branch;
-  //bool corMetType1Mcone5_isLoaded;
-  //vector<reco::CaloMET>	corMetType1Mcone7_;
-  //TBranch *corMetType1Mcone7_branch;
-  //bool corMetType1Mcone7_isLoaded;
+	vector<reco::CaloMET>	corMetType1Icone5_;
+	TBranch *corMetType1Icone5_branch;
+	bool corMetType1Icone5_isLoaded;
+	vector<reco::CaloMET>	corMetType1Mcone5_;
+	TBranch *corMetType1Mcone5_branch;
+	bool corMetType1Mcone5_isLoaded;
+	vector<reco::CaloMET>	corMetType1Mcone7_;
+	TBranch *corMetType1Mcone7_branch;
+	bool corMetType1Mcone7_isLoaded;
 	unsigned int	evt_nalltrkjets_;
 	TBranch *evt_nalltrkjets_branch;
 	bool evt_nalltrkjets_isLoaded;
@@ -1788,14 +1788,14 @@
 	if(hyp_other_jets_tq_jet_p4_branch == 0 ) {
 	cout << "Branch hyp_other_jets_tq_jet_p4 does not exist." << endl;
 	}
-	//TriggerResults_branch = 0;
-	//if (tree->GetAlias("TriggerResults") != 0) {
-	//	TriggerResults_branch = tree->GetBranch(tree->GetAlias("TriggerResults"));
-	//	TriggerResults_branch->SetAddress(&TriggerResults_);
-	//}
-	//if(TriggerResults_branch == 0 ) {
-	//cout << "Branch TriggerResults does not exist." << endl;
-	//}
+	TriggerResults_branch = 0;
+	if (tree->GetAlias("TriggerResults") != 0) {
+		TriggerResults_branch = tree->GetBranch(tree->GetAlias("TriggerResults"));
+		TriggerResults_branch->SetAddress(&TriggerResults_);
+	}
+	if(TriggerResults_branch == 0 ) {
+	cout << "Branch TriggerResults does not exist." << endl;
+	}
 	gen_met_branch = 0;
 	if (tree->GetAlias("gen_met") != 0) {
 		gen_met_branch = tree->GetBranch(tree->GetAlias("gen_met"));
@@ -4756,30 +4756,30 @@
 	if(hyp_trilep_jets_index_branch == 0 ) {
 	cout << "Branch hyp_trilep_jets_index does not exist." << endl;
 	}
-	//corMetType1Icone5_branch = 0;
-	//if (tree->GetAlias("corMetType1Icone5") != 0) {
-	//	corMetType1Icone5_branch = tree->GetBranch(tree->GetAlias("corMetType1Icone5"));
-	//	corMetType1Icone5_branch->SetAddress(&corMetType1Icone5_);
-	//}
-	//if(corMetType1Icone5_branch == 0 ) {
-	//cout << "Branch corMetType1Icone5 does not exist." << endl;
-	//}
-	//corMetType1Mcone5_branch = 0;
-	//if (tree->GetAlias("corMetType1Mcone5") != 0) {
-	//	corMetType1Mcone5_branch = tree->GetBranch(tree->GetAlias("corMetType1Mcone5"));
-	//	corMetType1Mcone5_branch->SetAddress(&corMetType1Mcone5_);
-	//}
-	//if(corMetType1Mcone5_branch == 0 ) {
-	//cout << "Branch corMetType1Mcone5 does not exist." << endl;
-	//}
-	//corMetType1Mcone7_branch = 0;
-	//if (tree->GetAlias("corMetType1Mcone7") != 0) {
-	//	corMetType1Mcone7_branch = tree->GetBranch(tree->GetAlias("corMetType1Mcone7"));
-	//	corMetType1Mcone7_branch->SetAddress(&corMetType1Mcone7_);
-	//}
-	//if(corMetType1Mcone7_branch == 0 ) {
-	//cout << "Branch corMetType1Mcone7 does not exist." << endl;
-	//}
+	corMetType1Icone5_branch = 0;
+	if (tree->GetAlias("corMetType1Icone5") != 0) {
+		corMetType1Icone5_branch = tree->GetBranch(tree->GetAlias("corMetType1Icone5"));
+		corMetType1Icone5_branch->SetAddress(&corMetType1Icone5_);
+	}
+	if(corMetType1Icone5_branch == 0 ) {
+	cout << "Branch corMetType1Icone5 does not exist." << endl;
+	}
+	corMetType1Mcone5_branch = 0;
+	if (tree->GetAlias("corMetType1Mcone5") != 0) {
+		corMetType1Mcone5_branch = tree->GetBranch(tree->GetAlias("corMetType1Mcone5"));
+		corMetType1Mcone5_branch->SetAddress(&corMetType1Mcone5_);
+	}
+	if(corMetType1Mcone5_branch == 0 ) {
+	cout << "Branch corMetType1Mcone5 does not exist." << endl;
+	}
+	corMetType1Mcone7_branch = 0;
+	if (tree->GetAlias("corMetType1Mcone7") != 0) {
+		corMetType1Mcone7_branch = tree->GetBranch(tree->GetAlias("corMetType1Mcone7"));
+		corMetType1Mcone7_branch->SetAddress(&corMetType1Mcone7_);
+	}
+	if(corMetType1Mcone7_branch == 0 ) {
+	cout << "Branch corMetType1Mcone7 does not exist." << endl;
+	}
 	evt_nalltrkjets_branch = 0;
 	if (tree->GetAlias("evt_nalltrkjets") != 0) {
 		evt_nalltrkjets_branch = tree->GetBranch(tree->GetAlias("evt_nalltrkjets"));
@@ -4969,7 +4969,7 @@
 		hyp_other_jets_tq_genPartonMother_p4_isLoaded = false;
 		hyp_other_jets_tq_genParton_p4_isLoaded = false;
 		hyp_other_jets_tq_jet_p4_isLoaded = false;
-		//TriggerResults_isLoaded = false;
+		TriggerResults_isLoaded = false;
 		gen_met_isLoaded = false;
 		gen_metPhi_isLoaded = false;
 		evt_CSA07FilterEff_isLoaded = false;
@@ -5340,9 +5340,9 @@
 		hyp_other_jets_tq_partonFlavour_isLoaded = false;
 		hyp_quadlep_jets_index_isLoaded = false;
 		hyp_trilep_jets_index_isLoaded = false;
-		//corMetType1Icone5_isLoaded = false;
-		//corMetType1Mcone5_isLoaded = false;
-		//corMetType1Mcone7_isLoaded = false;
+		corMetType1Icone5_isLoaded = false;
+		corMetType1Mcone5_isLoaded = false;
+		corMetType1Mcone7_isLoaded = false;
 		evt_nalltrkjets_isLoaded = false;
 		evt_nels_isLoaded = false;
 		evt_event_isLoaded = false;
@@ -5856,15 +5856,15 @@
 		}
 		return hyp_other_jets_tq_jet_p4_;
 	}
-  //edm::TriggerResults &TriggerResults()
-  //{
-  //	if (not TriggerResults_isLoaded) {
-  //		if (TriggerResults_branch != 0) 
-  //			TriggerResults_branch->GetEntry(index);
-  //		TriggerResults_isLoaded = true;
-  //	}
-  //	return TriggerResults_;
-  //}
+	edm::TriggerResults &TriggerResults()
+	{
+		if (not TriggerResults_isLoaded) {
+			if (TriggerResults_branch != 0) 
+				TriggerResults_branch->GetEntry(index);
+			TriggerResults_isLoaded = true;
+		}
+		return TriggerResults_;
+	}
 	float &gen_met()
 	{
 		if (not gen_met_isLoaded) {
@@ -9195,33 +9195,33 @@
 		}
 		return hyp_trilep_jets_index_;
 	}
-  //vector<reco::CaloMET> &corMetType1Icone5()
-  //{
-  //	if (not corMetType1Icone5_isLoaded) {
-  //		if (corMetType1Icone5_branch != 0) 
-  //			corMetType1Icone5_branch->GetEntry(index);
-  //		corMetType1Icone5_isLoaded = true;
-  //	}
-  //	return corMetType1Icone5_;
-  //}
-  //vector<reco::CaloMET> &corMetType1Mcone5()
-  //{
-  //	if (not corMetType1Mcone5_isLoaded) {
-  //		if (corMetType1Mcone5_branch != 0) 
-  //			corMetType1Mcone5_branch->GetEntry(index);
-  //		corMetType1Mcone5_isLoaded = true;
-  //	}
-  //	return corMetType1Mcone5_;
-  //}
-  //vector<reco::CaloMET> &corMetType1Mcone7()
-  //{
-  //	if (not corMetType1Mcone7_isLoaded) {
-  //		if (corMetType1Mcone7_branch != 0) 
-  //			corMetType1Mcone7_branch->GetEntry(index);
-  //		corMetType1Mcone7_isLoaded = true;
-  //	}
-  //	return corMetType1Mcone7_;
-  //}
+	vector<reco::CaloMET> &corMetType1Icone5()
+	{
+		if (not corMetType1Icone5_isLoaded) {
+			if (corMetType1Icone5_branch != 0) 
+				corMetType1Icone5_branch->GetEntry(index);
+			corMetType1Icone5_isLoaded = true;
+		}
+		return corMetType1Icone5_;
+	}
+	vector<reco::CaloMET> &corMetType1Mcone5()
+	{
+		if (not corMetType1Mcone5_isLoaded) {
+			if (corMetType1Mcone5_branch != 0) 
+				corMetType1Mcone5_branch->GetEntry(index);
+			corMetType1Mcone5_isLoaded = true;
+		}
+		return corMetType1Mcone5_;
+	}
+	vector<reco::CaloMET> &corMetType1Mcone7()
+	{
+		if (not corMetType1Mcone7_isLoaded) {
+			if (corMetType1Mcone7_branch != 0) 
+				corMetType1Mcone7_branch->GetEntry(index);
+			corMetType1Mcone7_isLoaded = true;
+		}
+		return corMetType1Mcone7_;
+	}
 	unsigned int &evt_nalltrkjets()
 	{
 		if (not evt_nalltrkjets_isLoaded) {
